### PR TITLE
F#25 lectorium chair

### LINKED ISSUE #25: Upload models of lectorium_chair
Upload blender models of lectorium_chair box into robert_gazebo/cad/ directory

Upload the visual models of lectorium_chairin .dae extension into robert_gazebo/meshes/ directory

Upload the collision models of lectorium_chairin .dae extension into robert_gazebo/meshes/ directory

Upload models of lectorium_chair in .sdf extension into robert_gazebo/models/lectorium_chair/ directory

Upload models of lectorium_chair in .config extension into robert_gazebo/models/lectorium_chair/ directory

### PR DESCRIPTION
**Pull request by LAnastasiia**
closes #25

### CODE REVIEW

File: robert_gazebo/models/lectorium_chair_orange/model.sdf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0'?>
+<sdf version='1.5'>
+  <model name="locker_008">
```
Comment (AndriyPt) [inline robert_gazebo/models/lectorium_chair_orange/model.sdf]: wrong model name :)

File: robert_gazebo/worlds/office.world [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" ?>
+office.world<?xml version="1.0" ?>
```
Comment (AndriyPt) [inline robert_gazebo/worlds/office.world]: please remove redundant text

**Review by AndriyPt — CHANGES_REQUESTED** [review]
please remove redundant mesges

**Comment by LAnastasiia** [comment]
Done

**Comment by LAnastasiia** [comment]
Done.

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Please revert all your changes in office.world file

**Comment by LAnastasiia** [comment]
Done.

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**